src/chrome/content/rules/Weibo.com-mixedcontent.xml

```diff
@@ -1,32 +0,0 @@
-<!--
-	For rules not causing mixed content, see Weibo.com.xml.
-
-
-	Mixed css on weihr.sinaapp.com from img.t.sinajs.cn
-
--->
-<ruleset name="Weibo.com (mixed content)" platform="mixedcontent">
-
-	<!--	Special cases:
-				-->
-
-	<target host="widget.weibo.com" />
-	
-	<!-- 504 Time out:
-	<target host="food.weibo.com" /-->
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^weihr\.sinaapp\.com$" name="^saeut$" /-->
-
-	<securecookie host="^\w" name="." />
-	
-<!--	Totally different site
-	<target host="hr.weibo.com" />
-	<rule from="^http://hr\.weibo\.com/"
-		to="https://weihr.sinaapp.com/" />
-				-->
-		
-	<rule from="^http:" to="https:" />
-
-</ruleset>
```

Comment:
I'd keep this mixedcontent platform for tor user,

Add targets:
- [ ] chart.weibo.com
- [ ] e.weibo.com